### ISSUE TITLE
Guest login

### ISSUE THREAD

**Issue by cchbr** [RocketChat/Rocket.Chat#10063]
I can't seem to find a way to offer guest login at the register page. The only way I've managed to activate guest login is to open the channels for public viewing. 

Is this the only way given the current version or can we allow guests to login and post within opening the chat to the non logged in public?

Thank you in advance and thank you for a great project!

**Comment by DoM1niC** [comment]
I think you mean anonymous login? Yeah thats will be great...

**Comment by cchbr** [comment]
Yes probably, although I'm not entirely sure what the difference between the two is.

**Comment by JSzaszvari** [comment]
Sorry I don't understand. 

Anonymous login = anyone can connect and post 

If the 'guests' have to 'login and post' then that's just a standard user who has registered ?

**Comment by cchbr** [comment]
Not necessarily. A registered user has to input an e-mail, password etc and register. Whilst a guest should only input a username at most, which is then discarded when the user leaves, at least that's how I view a guest user.

What is the definition of a guest on Rocket chat? Is it a user who's not logged in at all?

**Comment by cchbr** [comment]
The problem with anonymous login is that you have to open the entire chat to the public to activate it. Everything will probably get indexed by search engines and the users in the chat have no idea how many people are viewing what they're writing.
